gammapy/detect/test_statistics.py

```diff
@@ -72,30 +93,130 @@ def f_cash(x, counts, background, model):
     ----------
     x : float
         Model amplitude.
-    counts : `~numpy.ndarray`
+    counts : array
         Count map slice, where model is defined.
-    background : `~numpy.ndarray`
+    background : array
         Background map slice, where model is defined.
-    model : `~numpy.ndarray`
+    model : array
         Source template (multiplied with exposure).
     """
     with np.errstate(invalid='ignore', divide='ignore'):
         return cash(counts, background + x * FLUX_FACTOR * model).sum()
 
 
-def compute_ts_map(counts, background, exposure, kernel, flux=None,
-                   method='root', optimizer='Brent', parallel=True,
+def compute_ts_map_multiscale(maps, psf_parameters, scales=[0], downsample='auto',
+                              residual=False, morphology='Gaussian2D', width=0.3,
```

Comment:
The width parameter is not documented.
Probably 0.3 is something specific to your application ... if so this parameter should not have a default.

You should also say what happens to args and kwargs in a sentence ... if those are really needed ... it usually makes for complex code to be flexible in that way.